nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -162,6 +164,11 @@ In addition to numerous new and upgraded packages, this release has the followin
 
 - `idris2` now requires `--package` when using packages `contrib` and `network`, while previously these idris2 packages were automatically loaded.
 
+- `multimc` has been replaced with `multimc-bin`, which is a self-updating
+  package that uses official binaries from upstream. Users with existing
+  installations need not take any steps, as the new package stores data in the
+  same locations as the previous one.
+
```

Comment:
```suggestion
- Due to upstream FOSS complications `multimc` has been moved to `multimc-bin`, which is a self-updating package that uses official, but unreproducible binaries. PolyMC is a fork that does not have MultiMC's FOSS issues and does not self update. Users with existing installations who want to switch to PolyMC should rename `~/.local/share/multimc` to `~/.local/share/polymc`. The main config file's path has also moved from `~/.local/share/multimc/multimc.cfg` to `~/.local/share/polymc/polymc.cfg`.
```

Reply:
I think "upstream FOSS complications" seems a little vague. There should probably be some reasoning stated, but I'm not sure exactly what to say that would be appropriate for a changelog.

Reply:
upstream FOSS complications could be a link to the discussion that led to remove MultiMC from nixpkgs and also this PR if needed at all.